<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>566</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Collatinus</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../collatinus.qrc">
    <normaloff>:/images/collatinus.png</normaloff>:/images/collatinus.png</iconset>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout">
    <item>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QSplitter" name="splitter">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <property name="handleWidth">
         <number>20</number>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QTextEdit" name="EditLatin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="cursor" stdset="0">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="statusTip">
          <string/>
         </property>
        </widget>
        <widget class="QTabWidget" name="tabWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>textus</string>
          </attribute>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QTextEdit" name="EditTextus">
             <property name="autoFormatting">
              <set>QTextEdit::AutoAll</set>
             </property>
             <property name="readOnly">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <attribute name="title">
           <string>html</string>
          </attribute>
          <layout class="QVBoxLayout">
           <item>
            <widget class="QTextEdit" name="EditHtml"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>LaTeX</string>
          </attribute>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QTextEdit" name="EditLaTeX"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>flexio</string>
          </attribute>
          <layout class="QHBoxLayout">
           <item>
            <widget class="QTextEdit" name="EditFlexio">
             <property name="font">
              <font>
               <family>Monospace</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Auxilium">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAuxilium"/>
    <addaction name="separator"/>
    <addaction name="actionDe_Collatino"/>
   </widget>
   <widget class="QMenu" name="menuCapsa">
    <property name="title">
     <string>&amp;Capsa</string>
    </property>
    <addaction name="action_Noua"/>
    <addaction name="action_Onerare"/>
    <addaction name="action_Scribere"/>
    <addaction name="actionScribere_s_ub"/>
    <addaction name="separator"/>
    <addaction name="action_Exire"/>
   </widget>
   <widget class="QMenu" name="menu_Editio">
    <property name="title">
     <string>&amp;Editio</string>
    </property>
    <addaction name="actionInuenire"/>
    <addaction name="actionInuenire_etiam"/>
    <addaction name="separator"/>
    <addaction name="actionL_emmata_radere"/>
   </widget>
   <widget class="QMenu" name="menu_Visio">
    <property name="title">
     <string>&amp;Visio</string>
    </property>
    <addaction name="actionMaiores_litteras"/>
    <addaction name="actionMinores_litteras"/>
    <addaction name="actionSicut_est"/>
   </widget>
   <widget class="QMenu" name="menu_Lemmata">
    <property name="title">
     <string>&amp;Lemmata</string>
    </property>
    <addaction name="actionOmnia_lemmatizare"/>
    <addaction name="separator"/>
    <addaction name="actionAlphabetice"/>
    <addaction name="separator"/>
    <addaction name="actionSyntaxis"/>
    <addaction name="actionCum_textus_uocibus"/>
    <addaction name="separator"/>
    <addaction name="actionGermanice"/>
    <addaction name="actionAnglice"/>
    <addaction name="actionHispanice"/>
    <addaction name="actionGallice"/>
   </widget>
   <addaction name="menuCapsa"/>
   <addaction name="menu_Editio"/>
   <addaction name="menu_Visio"/>
   <addaction name="menu_Lemmata"/>
   <addaction name="menu_Auxilium"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Noua"/>
   <addaction name="action_Onerare"/>
   <addaction name="action_Scribere"/>
   <addaction name="actionInuenire"/>
   <addaction name="actionSyntaxis"/>
   <addaction name="actionOmnia_lemmatizare"/>
   <addaction name="actionAlphabetice"/>
   <addaction name="actionL_emmata_radere"/>
   <addaction name="actionMaiores_litteras"/>
   <addaction name="actionMinores_litteras"/>
   <addaction name="separator"/>
   <addaction name="action_Exire"/>
  </widget>
  <action name="action_Exire">
   <property name="icon">
    <iconset>
     <normaloff>:exit.svg</normaloff>:exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Exire</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="action_Noua">
   <property name="icon">
    <iconset>
     <normaloff>:new.svg</normaloff>:new.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Noua</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_Onerare">
   <property name="icon">
    <iconset>
     <normaloff>:open.svg</normaloff>:open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Onerare</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Scribere">
   <property name="icon">
    <iconset>
     <normaloff>:save.svg</normaloff>:save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Scribere</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionScribere_s_ub">
   <property name="text">
    <string>Scribere &amp;ut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionL_emmata_radere">
   <property name="icon">
    <iconset>
     <normaloff>:clear.svg</normaloff>:clear.svg</iconset>
   </property>
   <property name="text">
    <string>L&amp;emmata radere</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionMaiores_litteras">
   <property name="icon">
    <iconset>
     <normaloff>:font-size-up.svg</normaloff>:font-size-up.svg</iconset>
   </property>
   <property name="text">
    <string>Maiores litteras</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionMinores_litteras">
   <property name="icon">
    <iconset>
     <normaloff>:font-size-down.svg</normaloff>:font-size-down.svg</iconset>
   </property>
   <property name="text">
    <string>Minores litteras</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionSicut_est">
   <property name="text">
    <string>Sicut est</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionOmnia_lemmatizare">
   <property name="icon">
    <iconset>
     <normaloff>:lemmatiser.svg</normaloff>:lemmatiser.svg</iconset>
   </property>
   <property name="text">
    <string>Omnia lemmatizare</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionLineam">
   <property name="text">
    <string>Lineam</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionUocem">
   <property name="text">
    <string>Uocem</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionDe_Collatino">
   <property name="text">
    <string>de Collatino</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionAlphabetice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:alpha.svg</normaloff>:alpha.svg</iconset>
   </property>
   <property name="text">
    <string>Alphabetice</string>
   </property>
  </action>
  <action name="actionAuxilium">
   <property name="icon">
    <iconset>
     <normaloff>:help.svg</normaloff>:help.svg</iconset>
   </property>
   <property name="text">
    <string>Auxilium</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionLemmatiseMot">
   <property name="text">
    <string>LemmatiseMot</string>
   </property>
  </action>
  <action name="actionGermanice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Germanice</string>
   </property>
  </action>
  <action name="actionAnglice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Anglice</string>
   </property>
  </action>
  <action name="actionGallice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gallice</string>
   </property>
  </action>
  <action name="action_Quaestus">
   <property name="text">
    <string>&amp;Quaestus</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionCum_textus_uocibus">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>cum textus uocibus</string>
   </property>
  </action>
  <action name="actionInuenire">
   <property name="icon">
    <iconset>
     <normaloff>:find.svg</normaloff>:find.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Inuenire</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionInuenire_etiam">
   <property name="text">
    <string>Inuenire &amp;denuo</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionSyntaxis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:syntaxe.svg</normaloff>:syntaxe.svg</iconset>
   </property>
   <property name="text">
    <string>syntaxis</string>
   </property>
  </action>
  <action name="actionVide_texte">
   <property name="text">
    <string>vide_texte</string>
   </property>
  </action>
  <action name="actionHispanice">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Hispanice</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../collatinus.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Exire</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMaiores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditLatin</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditLatin</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMaiores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditTextus</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMaiores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditLaTeX</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMaiores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditFlexio</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditFlexio</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditLaTeX</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>61</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditTextus</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMinores_litteras</sender>
   <signal>triggered()</signal>
   <receiver>EditHtml</receiver>
   <slot>zoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMaiores_litteras</sender>
   <signal>triggered(bool)</signal>
   <receiver>EditHtml</receiver>
   <slot>zoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
